cpp/include/cudf/detail/transform.hpp

```diff
@@ -24,7 +24,7 @@ namespace detail {
 /**
  * @copydoc cudf::experimental::transform
  *
- * @param stream        CUDA stream on which to execute kernels
+ * @param stream        CUDA stream used for device memory operations and kernel launches.
```

Comment:
Is this spacing intentional ?

Reply:
no. I simply replaced existing text. @devavret replaced copydoc and left this spacing as it is. 
This spacing may not be needed since there is no alignment now.